apps/financial-aid/web-veita/src/components/Nav/Nav.tsx

```diff
@@ -57,9 +58,15 @@ const Nav = ({ showInMobile }: Props) => {
           </Box>
 
           <Box paddingLeft={2} className={'headLine'}>
-            <Text as="h1" lineHeight="sm">
-              <strong>Veita</strong> • Umsóknir um fjárhagsaðstoð
-            </Text>
+            {admin ? (
+              <Text as="h1" lineHeight="sm">
+                <strong>Mamma Sveitó</strong> • Umsjón með sveitarfélögum
```

Comment:
Are we sure about this name outwards...... Thought it is more of a internal naming.....